Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -81,6 +82,18 @@ void SetStartingPoint(CGPoint newPoint) {
         startingPoint = newPoint;
     }
 
+    void SetLastTransform(const CGAffineTransform* transform) {
+        if (transform) {
+            lastTransform = transform;
+        } else {
+            lastTransform = &CGAffineTransformIdentity;
+        }
+    }
+
+    const CGAffineTransform* GetLastTransform() {
```

Comment:
by that I mean GetLastTransform() const {}

---
In reply to: [88733190](https://github.com/Microsoft/WinObjC/pull/1384#discussion_r88733190) [](ancestors = 88733190)